libraries/manifest-validation/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{

```

Comment:
You could simplify this to keep it simple and like the other projects, similar to https://github.com/pwa-builder/PWABuilder/blob/main/libraries/site-analytics/tsconfig.json

Reply:
Shouldn't the built directory still be in ./dist and not in the root dist though?

Reply:
yeah, I meant most of the config can be removed, but not all. The outDir and rootDir probably will need to stay (maybe few others)